### PR TITLE
Add page on installation name selection

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/roadmap/issues/1353

This PR adds a new documentation page for customers regarding selecting an installation name.

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Review by anvddriesch — APPROVED** [review]

**Comment by marians** [comment]
Thanks! Also happy for a native speaker check. Looking at @kuosandys :)

File: src/content/getting-started/installation-name/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+last_review_date: 2022-09-08
+---
+
+A Giant Swarm installation includes a management cluster, bastion hosts, the workload clusters (depending on your viewpoint) and a few more components specific to the cloud provider used.
+
+## Why an installation name
+
+Every Giant Swarm installations has a unique name. This name is part of the DNS zone used for all endpoints of this installation.
+
+When we set up a new installation for you, we want you to pick a name for it. Think of this name as your handy alias for everything you do with Giant Swarm in a specific cloud provider region, e. g. _Google Cloud Finland (europe-north1)_.
```
Comment (kuosandys) [inline src/content/getting-started/installation-name/index.md:22]:
```suggestion
When we set up a new installation for you, we want you to pick a name for it. Think of this name as your handy alias for everything you do with Giant Swarm in a specific cloud provider region, e.g. _Google Cloud Finland (europe-north1)_.
```

File: src/content/getting-started/installation-name/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Once a name is decided on, you will use it often on your communication with Giant Swarm. Of course, we encourage you to also use it in internal communication.
```
Comment (kuosandys) [inline src/content/getting-started/installation-name/index.md:24]:
```suggestion
Once a name is decided on, you will use it often in your communication with Giant Swarm. Of course, we encourage you to also use it in internal communication.
```

File: src/content/getting-started/installation-name/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Criteria for good installation names
+
+Besides being usable in DNS, there are many criteria which distinguish a good installation name. It should be easy to pronounce in an english language context, easy to type, and ideally easy to memorize. On the other hand, it should not reveal information about you as a customer.
```
Comment (kuosandys) [inline src/content/getting-started/installation-name/index.md:28]:
```suggestion
Besides being usable in DNS, there are many criteria which distinguish a good installation name. It should be easy to pronounce in an English language context, easy to type, and ideally easy to memorize. On the other hand, it should not reveal information about you as a customer.
```

File: src/content/getting-started/installation-name/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Installation names cannot be changed. So the best names are long-lasting, even in the case of context changes. For example, changing the purpose of the installation, or even a company name change.
+
+We provide a GitHub repository with [installation name candidates](https://github.com/giantswarm/installation-names) available for you to pick from. The names offered there should fulfill all of our criteria for a name.
+
+## How to select a name
+
+To select a name for your new installation, please
+
+1. Browse the available names in the `.txt` files within the [installation-names](https://github.com/giantswarm/installation-names) repository and decide for a name. Feel free to pick the one you like best, for whatever reason.
```
Comment (kuosandys) [inline src/content/getting-started/installation-name/index.md:38]:
```suggestion
1. Browse the available names in the `.txt` files within the [installation-names](https://github.com/giantswarm/installation-names) repository and decide on a name. Feel free to pick the one you like best, for whatever reason.
```

**Comment by marians** [comment]
Thanks @kuosandys for the language touch-up!

**Pull request merged**